src/SDPoptimize.jl

```diff
@@ -632,6 +637,7 @@ the alea realization
 
 Returns :
 - V(x0) (Float64)
+
 """
 function get_control(model::SPModel,param::SDPparameters,V::Array{Float64}, t::Int64, x::Array, w::Array)
 
```

Comment:
We should check if we are in the HD case, if not warn("the problem is defined in decision-hazard and control is obtained assuming hazard-decision structure")